### PR TITLE
Changelog 145

### PR DESCRIPTION
**Pull request by mchernyavsky**

### CODE REVIEW

File: _posts/2021-04-12-changelog-145.markdown [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* Basic multiresolution at module scope in new [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7005 -->
+* Annotate wrong order of const generic parameters without `const_generics` feature
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6457 -->
+* Add inspection that annotates invalid references in `TOML` string literals. The inspection can be extended by other plugins. For example, Rust plugin provides file references to local dependency, build script and workspace paths in `Cargo.toml` as well as references to Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html) and the inspection highlights all unresolved ones. Additionally, it provides a quick-fix to create a non-existent file from the corresponding file reference (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6969 -->
+* Make dependencies completion in `Cargo.toml` more intuitive and faster when using crates local index. Note, this feature is disabled by default for now. To use it, enable `org.rust.crates.local.index` experimental feature
```
Comment (Undin) [inline _posts/2021-04-12-changelog-145.markdown:25]: let's move this item into fix section and group it with other fixes related to crates local index

File: _posts/2021-04-12-changelog-145.markdown [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* Don't import default type parameters by `Extract Function` refactoring (`Refactor | Extract | Method` or <kbd>Cmd</kbd> + <kbd>Alt</kbd> + <kbd>M</kbd>)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7026 -->
+* Omit default type parameters in a `Convert to type` quick-fix text
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6945 -->
+* Fix exception in `Move` refactoring (`Refactor | Move` or <kbd>F6</kbd>) when it was first cancelled and then invoked again
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6968 -->
+* Don't suggest postfix templates inside string literals
```
Comment (Undin) [inline _posts/2021-04-12-changelog-145.markdown:47]:
```suggestion
* Don't suggest [postfix templates](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-generation.html#postfix-completion) inside string literals
```

File: _posts/2021-04-12-changelog-145.markdown [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7033 -->
+* Basic multiresolution at module scope in new [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7005 -->
+* Annotate wrong order of const generic parameters without `const_generics` feature
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6457 -->
+* Add inspection that annotates invalid references in `TOML` string literals. The inspection can be extended by other plugins. For example, Rust plugin provides file references to local dependency, build script and workspace paths in `Cargo.toml` as well as references to Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html) and the inspection highlights all unresolved ones. Additionally, it provides a quick-fix to create a non-existent file from the corresponding file reference (by [@Kobzol])
```
Comment (ortem) [inline _posts/2021-04-12-changelog-145.markdown:22]: IMO this description is a bit difficult to read right now, maybe rephrase it somehow? A possible option:
```suggestion
* Add `Unresolved reference` inspection for `Cargo.toml` files. The inspection highlights unresolved references in `TOML` string literals and provides a quick-fix to create a new file for the corresponding unresolved path.
Note, the plugin provides such references for local dependencies, build scripts, workspace paths, and Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html). The set of analyzed references can be extended by other plugins (by [@Kobzol])
```

File: _posts/2021-04-12-changelog-145.markdown [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+* Don't suggest postfix templates inside string literals
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7013 -->
+* Fix type inference for array index operator in Rust 1.50
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7007 -->
+* Fix unification with the never type `!` in `match` arms
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7053 -->
+* Fix name resolution of items in standard library with `cfg` attribute in some cases
```
Comment (ortem) [inline _posts/2021-04-12-changelog-145.markdown:56]:
```suggestion
* Fix name resolution of stdlib items with `cfg` attributes in some cases
```

**Comment by mchernyavsky** [comment]
@Undin Fixed.

File: Gemfile.lock [hunk 5]
```diff
@@ -1,238 +0,0 @@
-GEM
```
Comment (Undin) [inline Gemfile.lock:1]: Why did you remove it?

Comment (mchernyavsky) [reply Gemfile.lock]: Fixed.

**Review by Undin — APPROVED** [review]
LGTM
